istioctl/cmd/istioctl/main.go

```diff
@@ -656,18 +676,123 @@ func printShortOutput(writer io.Writer, _ model.ConfigStore, configList []model.
 
 	var w tabwriter.Writer
 	w.Init(writer, 10, 4, 3, ' ', 0)
-	fmt.Fprintf(&w, "NAME\tKIND\tNAMESPACE\n")
+	prevType := ""
+	var outputter func(model.Config, io.Writer)
 	for _, c := range configList {
-		kind := fmt.Sprintf("%s.%s.%s",
-			crd.KabobCaseToCamelCase(c.Type),
-			c.Group,
-			c.Version,
-		)
-		fmt.Fprintf(&w, "%s\t%s\t%s\n", c.Name, kind, c.Namespace)
+		if prevType != c.Type {
+			if prevType != "" {
+				// Place a newline between types when doing 'get all'
+				fmt.Fprintf(&w, "\n")
+			}
+			heading, ok := shortOutputHeadings[c.Type]
+			if !ok {
+				heading = unknownShortOutputHeading
+			}
+			fmt.Fprintf(&w, "%s\n", heading)
+			prevType = c.Type
+
+			if outputter, ok = shortOutputters[c.Type]; !ok {
```

Comment:
super nit: `ok` -> `exists`

Reply:
I am happy to make that change ... but it doesn't match the rest of Istio.

Counting usage, I see 
7.7k for “, ok :=“ and 325 for “, ok =“
vs
185 for “, exists :=“ and 9 for “, exists =“

Reply:
I'm not trying to pick a fight.  498 matches in Istio for the for regex `, ok :?= [a-zA-Z]+\[`, 36 for the regex `, exists :?= [a-zA-Z]+\[/`

Reply:
`ok` is fine. https://github.com/golang/go/wiki/CodeReviewComments#in-band-errors and https://golang.org/doc/effective_go.html#maps both use `ok` in their examples to test existance.